Makefile

```diff
@@ -372,14 +374,20 @@ credentials/pull-secret.json:
 
 .PHONY: lifecycle-agent-deploy
 lifecycle-agent-deploy: lifecycle-agent
-	KUBECONFIG=../$(SNO_KUBECONFIG) \
-	IMG=$(LCA_IMAGE) \
-		make -C lifecycle-agent install deploy
-	@echo "Waiting for deployment lifecycle-agent-controller-manager to be available"; \
-	until $(oc) wait deployment -n openshift-lifecycle-agent lifecycle-agent-controller-manager --for=condition=available=true; do \
-		echo -n .;\
-		sleep 5; \
-	done; echo
+	@if [ -n $(LCA_OPERATOR_BUNDLE_IMAGE) ]; then \
+		KUBECONFIG=../$(SNO_KUBECONFIG) \
+		BUNDLE_IMG=$(LCA_OPERATOR_BUNDLE_IMAGE) \
+			make -C lifecycle-agent bundle-run ;\
+	else \
+		KUBECONFIG=../$(SNO_KUBECONFIG) \
```

Comment:
Unfortunately, this leads to the following error:

```log
/bin/sh: -c: line 1: syntax error near unexpected token `then'
/bin/sh: -c: line 1: `if [ -n """" ]; then \'
make: *** [Makefile:378: lifecycle-agent-deploy] Error 1
```

I couldn't find a good way to fix that (apart from export-ing KUBECONFIG). WDYT?

Reply:
Exporting is also fine I guess

Reply:
> Oh it's probably the @

That's what I thought, but even when I removed it the same error occurred. I'll give it another try or just export it.

Reply:
Actually, it's both things, your change not only changes the variable definition, but the way it is defined:

```
KUBECONFIG=blablabla command
```

vs

```
KUBECONFIG=blablabla
command
```

so, it needs to be exported, and it also needs the ; in the end:

```
export KUBECONFIG=blablabla;
if blablablabla
```

Adding the @ at the beginning of the command block is also handy, since then `make` will be less verbose and wont print the whole command when running it:

```
@export KUBECONFIG=blablabla;
if blablablabla
```

Reply:
Javi and I talked offline and the version above still doesn't resolve the issue, so I ended up `export-`ing the env var.

Reply:
Lol sorry for all this just wanted to make a quick suggestion :sweat_smile:

Reply:
It made sense and we ended up with the proper way ;)